pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -66,6 +67,17 @@ func (pl *CSILimits) Name() string {
 	return CSIName
 }
 
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable.
+func (pl *CSILimits) EventsToRegister() []framework.ClusterEvent {
+	return []framework.ClusterEvent{
+		{Resource: framework.CSINode, ActionType: framework.Add | framework.Update},
```

Comment:
Not quite sure if we should count regular `Node`, probably not as csi.go#getVolumeLimits() stated:

```
// TODO: stop getting values from Node object in v1.18
```